blueprints/custom-post/blueprints.json

```
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+    "meta": {
+        "title": "Custom Post Type: Books",
+        "description": "Blueprint that added a custom post type to playground",
+        "author": "bph",
+        "categories": ["Content", "CPT"]
+    },
+    "landingPage": "/wp-admin/",
+    "steps":[
+            {
+                "step": "login"
+            },
+            {
+                "step": "mkdir",
+                "path": "/wordpress/wp-content/plugins/books"
+            },
+            {
+                "step": "writeFile",
+                "path": "/wordpress/wp-content/plugins/books/books.php",
+                "data": {
+                    "resource": "url",
+                    "url": "https://raw.githubusercontent.com/adamziel/blueprints/custom-post/book.txt"
+                }
+            },
+            {
+                "step": "activatePlugin",
+                "pluginPath": "/wordpress/wp-content/plugins/books/"
```

This is a relative path to the `.php` file – it would be great to document that or rename the option to some succinct version of `relativePathToPluginPhpFile`:

```suggestion
                "pluginPath": "books/books.php"
```